pipeline_scripts/override_tclean_commands.json

```
@@ -14,11 +14,43 @@
     "tclean_cube_pars": {
       "spw24": {
         "imagename": "uid___A001_X15a0_X1a4.s38_0.Sgr_A_star_sci.spw24.cube.I.iter1.reclean",
-        "cyclefactor": 2.0
+        "cyclefactor": 2
       },
       "spw26": {
         "imagename": "uid___A001_X15a0_X1a4.s38_0.Sgr_A_star_sci.spw26.cube.I.iter1.reclean",
-        "cyclefactor": 2.0
+        "cyclefactor": 2
+      }
+    }
+  },
+  "Sgr_A_st_ao_03_TM1": {
+    "tclean_cube_pars": {
+      "spw35": {
+        "imagename": "uid___A001_X15a0_X190.s38_0.Sgr_A_star_sci.spw35.cube.I.iter0.reclean",
```

should this be `iter1`?

---

thanks for pointing the mistake.

---

you can just accept the suggestion:
```suggestion
        "imagename": "uid___A001_X15a0_X190.s38_0.Sgr_A_star_sci.spw35.cube.I.iter1.reclean",
```